omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -233,14 +196,29 @@ Locally built server
    :doc:`/developers/build-system`
         Developer documentation page on how to build the OMERO.server
 
-5. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/openmicroscopy/dist`.
-   If it is not already open, open your :file:`.bash_profile`::
+4. Once the build completes, the OMERO server build output will be located in :file:`openmicroscopy/dist`.
+
+
+OMERO configuration
+-------------------
+
+1. Open your :file:`.bash_profile` in a text editor, 
+   such as the built-in TextEdit app::
 
     $ open -a TextEdit.app ~/.bash_profile
 
-   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+   Add an environment variable :envvar:`OMERO_SERVER` to the :file:`.bash_profile` which points
+   to the location of the OMERO executable::
 
-    export PATH=~/Projects/Omero/code/openmicroscopy/dist/bin:$PATH
+    # Pre-built server...
+    export OMERO_SERVER=/path/to/OMERO.server-x.x.x-ice36-bxx
+    # ...OR locally built server
+    export OMERO_SERVER=/path/to/openmicroscopy/bin
```

Comment:
this line for locally built server does not look right (see the line which you deleted in this PR)
``export PATH=~/Projects/Omero/code/openmicroscopy/dist/bin:$PATH``
this should probably be replaced by
``export OMERO_SERVER=/path/to/openmicroscopy/dist``
(instead of ``bin`` at the end you need ``dist``)

Reply:
Ooops. yes. Good catch. Fixed.